python/ray/tests/test_basic_5.py

```diff
@@ -119,5 +120,24 @@ def test_internal_kv(ray_start_regular):
         kv._internal_kv_list("@namespace_abc", namespace="n")
 
 
+@pytest.mark.skipif(
+    os.environ.get("CI") and sys.platform != "linux",
+    reason="This test is only run on linux CI machines.")
+def test_using_hostname(ray_start_cluster):
```

Comment:
```suggestion
def test_ray_init_using_hostname(ray_start_cluster):
```
Maybe move this to `ray_init_test`?